api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java

```diff
@@ -2,34 +2,51 @@
 
 import javax.inject.Provider;
 import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.jdbc.ReportingQueryService;
 import org.pmiops.workbench.model.ReportingSnapshot;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
-/*
- * Calls the ReportingSnapshotService to obtain the application data from MySQL, Terra (soon),
- * and possibly other sources, then calls the uploadSnapshot() method on the configured ReportingUploadService
- * to upload to various tables in the BigQuery dataset. There is virtually no business logic in this class
- * except for the method for choosing the correct upload implementation.
+/**
+ * Calls the ReportingSnapshotService to obtain the application data from MySQL, Terra (soon), and
+ * possibly other sources, then calls the uploadSnapshot() method on the configured
+ * ReportingUploadService to upload to various tables in the BigQuery dataset. There is virtually no
+ * business logic in this class except for the method for choosing the correct upload
```

Comment:
you can delete this sentence. there's only one upload implementation now.

Note also that the ReportingQueryService has no dependency on BigQuery, and the ReportingUploadService has no dependency on MySQL, so in principle we could swap out the data source and destination independently.